index.js

```diff
@@ -22,6 +22,11 @@ module.exports = opts => buf => {
 		args.push(`-${opts.optimizationLevel}`);
 	}
 
+	if (typeof opts.iterations === 'number') {
+		args.push('-i');
+		args.push(opts.iterations);
```

Comment:
Use the long-flag:

```js
args.push('--iter', opts.iterations);
```

Reply:
Will do!  Just curious, what's the reason for this preference?  More understandable for casual reader?

Reply:
Improves readability

Reply:
> More understandable for casual reader?

Not just casual reader. For myself too. I don't go around and remember every possible short-flag.